### PR TITLE
Improve documentation for NDEFReadingEvent

### PR DESCRIPTION
**Pull request by jpmedley**
> Issue number that this PR fixes (if any). For example: 'Fixes #987654321'

NA

> What was wrong/why is this fix needed? (quick summary only)

This PR adds additional information related to the interface.

> Anything else that could help us review it

NA

### CODE REVIEW

**Comment by Elchi3** [comment]
So, this PR fails because the file and folder names aren't completely lowercase. I've been hit by this quite a few times, too. Not sure if tooling could be made better here.

**Comment by jpmedley** [comment]
Fixed.

File: files/en-us/web/api/ndefreadingevent/ndefreadingevent/index.html [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+</dl>
+
+
+<h2 id="Specifications">Specifications</h2>
+
+<p>{{Specifications}}</p>
+
+<h2 id="Browser_compatibility">Browser compatibility</h2>
+
+<div class="hidden">The compatibility table in this page is generated from structured data. If you'd like to contribute to the data, please check out <a href="https://github.com/mdn/browser-compat-data">https://github.com/mdn/browser-compat-data</a> and send us a pull request.</div>
```
Comment (sideshowbarker) [inline files/en-us/web/api/ndefreadingevent/ndefreadingevent/index.html:73]:
```suggestion
```

File: files/en-us/web/api/ndefreadingevent/message/index.html [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    console.log(serialNumber);
+  });</pre>
+
+<h2 id="Specifications">Specifications</h2>
+
+<p>{{Specifications}}</p>
+
+<h2 id="Browser_compatibility">Browser compatibility</h2>
+
+<div class="hidden">The compatibility table in this page is generated from structured data. If you'd like to contribute to the data, please check out <a href="https://github.com/mdn/browser-compat-data">https://github.com/mdn/browser-compat-data</a> and send us a pull request.</div>
```
Comment (sideshowbarker) [inline files/en-us/web/api/ndefreadingevent/message/index.html:52]:
```suggestion
```

File: files/en-us/web/api/ndefreadingevent/message/index.html [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
```
Comment (sideshowbarker) [inline files/en-us/web/api/ndefreadingevent/message/index.html:53]:
```suggestion
```

File: files/en-us/web/api/ndefreadingevent/ndefreadingevent/index.html [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+
```
Comment (sideshowbarker) [inline files/en-us/web/api/ndefreadingevent/ndefreadingevent/index.html:74]:
```suggestion
```

File: files/en-us/web/api/ndefreadingevent/serialNumber/index.html [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+  });</pre>
+
+<h2 id="Specifications">Specifications</h2>
+
+<p>{{Specifications}}</p>
+
+<h2 id="Browser_compatibility">Browser compatibility</h2>
+
+<div class="hidden">The compatibility table in this page is generated from structured data. If you'd like to contribute to the data, please check out <a href="https://github.com/mdn/browser-compat-data">https://github.com/mdn/browser-compat-data</a> and send us a pull request.</div>
+
```
Comment (sideshowbarker) [inline files/en-us/web/api/ndefreadingevent/serialNumber/index.html:52]:
```suggestion
```

**Review by sideshowbarker — APPROVED** [review]

**Pull request merged**